tensorflow/compiler/xla/service/cpu/compiler_functor.cc

```diff
@@ -211,6 +214,11 @@ void CompilerFunctor::AddOptimizationPasses(
   builder.SLPVectorize = opt_level_ > 1;
 
   builder.populateFunctionPassManager(*function_passes);
+
+  llvm::PassRegistry &Registry = *llvm::PassRegistry::getPassRegistry();
+  ::polly::initializePollyPasses(Registry);
+  ::polly::registerPollyPasses(*module_passes);
```

Comment:
This needs to be guarded by an off by default flag for now.

Google C++ style is `llvm::PassRegistry &Registry = *llvm::PassRegistry::getPassRegistry();`

Reply:
This hasn't been addressed.